tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -997,6 +998,14 @@ void ROOT::Experimental::Internal::RMiniFileReader::ReadBuffer(void *buffer, siz
 ////////////////////////////////////////////////////////////////////////////////
 
 
+Long64_t ROOT::Experimental::RNTuple::Merge(TCollection* /* input */, TFileMergeInfo* /* mergeInfo */) {
```

Comment:
Let's define the method in a new compilation unit (e.g. `RNTupleMerge`).  This should make it easier to separate the mini RNTuple library from the full version.

Reply:
Fixed in fef9e17732a2a87a6d22b57e60d8db1cd9eaf43f